modules/ducktests/tests/ignitetest/tests/calcite_test.py

```diff
@@ -0,0 +1,70 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This module contains smoke tests that checks that services work
+"""
+from ignitetest.services.ignite import IgniteService
+from ignitetest.services.ignite_app import IgniteApplicationService
+from ignitetest.services.utils.ignite_configuration import IgniteConfiguration, IgniteThinJdbcConfiguration
+from ignitetest.services.utils.ssl.client_connector_configuration import ClientConnectorConfiguration
+from ignitetest.utils import ignite_versions, cluster
+from ignitetest.utils.ignite_test import IgniteTest
+from ignitetest.utils.version import DEV_BRANCH, IgniteVersion
+
+from ignitetest.utils.bean import Bean
+
+
+class CalciteTest(IgniteTest):
+    """
+    Calcite engine tests
+    """
+    @cluster(num_nodes=2)
+    @ignite_versions(str(DEV_BRANCH))
+    def test_calcite_select_queries(self, ignite_version):
```

Comment:
I would emphasize that testing is done here 

1. Via the Thin JDBC driver (in contrast to native Ignite SQL API). 
2. Just operators are tested.

So name test say `test_std_sql_operators_via_thin_jdbc`